### ISSUE TITLE
Fix Value not stored / after power on VOltage setting wron

### ISSUE THREAD

**Issue by Maik2016** [mb-software/esphome-huawei-r4850#4]
Hello,

I set the Voltage to 55.5V and Current to 1A as default values for the start up or in case of CAN problems.

It looks like the Fixed savae function did not work for the current  it did not charge after restart of the power supply.  If I manual add 55.5V it starts intimidate

**Comment by jdrescher2006** [comment]
Yes I can confirm this bug. Current setting is not working in offline mode.
This results in that the power supply will output its maximum current on startup, which might be bad in certain situations...

**Comment by mager33** [comment]
Do these values survive cut of AC power?

**Comment by jdrescher2006** [comment]
No. When the device starts, it always starts with maximum voltage and current.

**Comment by jkoljo** [comment]
Can confirm the same issue with my R4850G2 + ESP32-EVB setup. Offline voltage value is not saved correctly. Current setting seems to be saved ok. Also, the current input field populates in Home Assistant upon startup, but voltage does not.
